src/ansible_builder/ee_schema.py

```diff
@@ -391,8 +392,8 @@ def validate_schema(ee_def: dict):
     if 'version' in ee_def:
         try:
             schema_version = int(ee_def['version'])
-        except ValueError:
-            raise DefinitionError(f"Schema version not an integer: {ee_def['version']}")
+        except ValueError as e:
+            raise DefinitionError(f"Schema version not an integer: {ee_def['version']}") from e
```

Comment:
Another unrelated change, but seems techincally correct to me. Is there a pylint check for this that we can enable?